src/encodings/encoding.rs

```diff
@@ -1149,6 +1150,26 @@ mod tests {
     );
   }
 
+  #[test]
+  fn test_delta_byte_array_reuse_encoder() {
```

Comment:
Would it be possible to name the test so it reflects the problem, e.g. test_delta_byte_array_reset_previous_value or something like that?

Reply:
Sure. I renamed the test to be `test_issue_47` and added a link of the issue to it. Let me know if you think this is OK.